src/com/javarush/task/task04/task0401/Solution.java

```diff
@@ -0,0 +1,39 @@
+package com.javarush.task.task04.task0401;
+
+/*
+Мне не подходит этот возраст…
+*/
+
+public class Solution {
+    public static void main(String[] args) {
+        Person person = new Person();
+        System.out.println("Age is: " + person.age);
+        person.adjustAge(person.age);
+        System.out.println("Adjusted age is: " + person.age);
+    }
+
+    public static class Person {
+        public int age = 20;
+
+        public void adjustAge(int age) {
+            this.age = age + 20;
```

Comment:
Почему +20?

Reply:
НЕ статическая переменная принадлежит объекту класса.

Reply:
```suggestion
            this.age += age;
```

Reply:
Строка выше равнозначна:
```suggestion
            this.age = this.age + age;
```